### ISSUE TITLE
Measuring Latency

### ISSUE THREAD

**Issue by alexfjw** [pyxyyy/gesture-drumkit#8]
Would be good if we could do this. 
Gives us more stuff to write in the report.

Depending on how the app works, we might need transmission latency.
We can estimate that from the RTT of sending a msg & waiting for acknowledgement.

- [ ] transmission latency
- [ ] recognition latency

**Comment by alexfjw** [comment]
We might want to send data in batches.
Right now, each message is received in an interval of 40ms.
However, the data is recorded 5ms apart. (5ms -> 200hz)

**Comment by alexfjw** [comment]
Measuring backpressure:
Get difference between time on phone when packet was received & time on watch when packet was created at t=1 & t=150.
If the difference increases at t=150 is larger than the difference at t=1, we have a bunch of messages queued to be sent. There is significant backpressure.

Turns out that there's negligible difference at t=150 & t=1 when we set message size per packet to 20. (~5ms) 
Therefore, 20 is the optimal size as of now, unless we use multi-threading to send packets.
